tasks/cassieq.rake

```diff
@@ -0,0 +1,87 @@
+require "yaml"
+require "curb"
+require "json"
+
+namespace :cassieq do
+  desc "Start a CassieQ container in development mode"
+  task :start_dev do
+    start_cassieq
+  end
+
+  desc "Run RSpec test suite againts CassieQ dev container"
+  task :test_client do
+    start_cassieq { run_tests_and_quit }
+  end
+end
+
+def start_cassieq
+  system "docker pull paradoxical/cassieq"
+
+  print "Starting CassieQ "
+  pid = fork { `docker run -it -p 8080:8080 -p 8081:8081 paradoxical/cassieq dev` }
+
+  until cassieq_ready?(host) do
```

Comment:
Love this! probably ought ot put a max time though so it doesn't spin forever if it ever breaks